### DISCUSSION TITLE
Es posible que mi hermano se una al curso y tener su correo de innovacion de igual manera que todos?

### DISCUSSION THREAD

**Discussion by anthonyrgc** [Launch-X-Latam/OnBoarding#16] [Q&A]
pregunto porque el no pudo hacer los 20 cursos de azure para ingresar a launch X y el quisiera hacer este curso de igual manera y poder tener un certificado

**Comment by JosafatJimenezB** [comment]
no fue necesario completar los cursos de azure para entrar aqui ya que es un programa distintio, pero aun asi que se rife en azure!!!

asi que si puede

**Reply by gumodi35** [reply]
Manda mensaje a contacto innovación, para que te comentén si es posible, por las fechas que ya pasaron ñ.
